{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "161023a9-6e4c-44ab-9f40-d783cccfa17f",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "import dask\n",
    "import dask.dataframe as dd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "82f01bb7-97fb-45a9-932b-9c24a6172ba8",
   "metadata": {},
   "outputs": [],
   "source": [
    "from IPython.display import Audio, display\n",
    "\n",
    "url_done = (\n",
    "    \"https://sound.peal.io/ps/audios/000/000/537/original/woo_vu_luvub_dub_dub.wav\"\n",
    ")\n",
    "url_done = \"https://www.myinstants.com/media/sounds/taco-bell-bong-sfx.mp3\"\n",
    "url_done = \"https://www.myinstants.com/media/sounds/magic_immune.mp3\"\n",
    "# url_done=\"https://www.myinstants.com/media/sounds/tindeck_1.mp3\"\n",
    "# url_done=\"https://www.myinstants.com/media/sounds/dun_dun_1.mp3\"\n",
    "\n",
    "\n",
    "def allDone():\n",
    "    display(Audio(url=url_done, autoplay=True,))\n",
    "\n",
    "\n",
    "url_exception = \"http://www.wav-sounds.com/movie/austinpowers.wav\"\n",
    "url_exception = \"https://www.myinstants.com/media/sounds/roblox-death-sound_1.mp3\"\n",
    "\n",
    "\n",
    "def play_sound(self, etype, value, tb, tb_offset=None):\n",
    "    self.showtraceback((etype, value, tb), tb_offset=tb_offset)\n",
    "    display(Audio(url=url_exception, autoplay=True))\n",
    "\n",
    "\n",
    "get_ipython().set_custom_exc((Exception,), play_sound)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "188bb35e-5303-408b-8b12-f1520393e831",
   "metadata": {},
   "outputs": [],
   "source": [
    "from dask.distributed import Client\n",
    "\n",
    "client = Client(n_workers=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2d46aebf-deb4-4f9a-885f-9c55a1f6a19f",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_dir = os.path.join(os.getcwd(), \"..\", \"data\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "92890f7a-eb99-46fd-962f-2de2cac8e0a2",
   "metadata": {},
   "outputs": [],
   "source": [
    "import datetime\n",
    "from functools import lru_cache\n",
    "\n",
    "\n",
    "def lower_case_sort_columns(df):\n",
    "    df.columns = df.columns.str.lower()\n",
    "\n",
    "    df = df[\n",
    "        [\n",
    "            \"arrive_time\",\n",
    "            \"data_source\",\n",
    "            \"direction\",\n",
    "            \"door\",\n",
    "            \"dwell\",\n",
    "            \"estimated_load\",\n",
    "            \"leave_time\",\n",
    "            \"lift\",\n",
    "            \"location_id\",\n",
    "            \"maximum_speed\",\n",
    "            \"offs\",\n",
    "            \"ons\",\n",
    "            \"pattern_distance\",\n",
    "            \"route_number\",\n",
    "            \"schedule_status\",\n",
    "            \"service_date\",\n",
    "            \"service_key\",\n",
    "            \"stop_time\",\n",
    "            \"train\",\n",
    "            \"train_mileage\",\n",
    "            \"trip_number\",\n",
    "            \"vehicle_number\",\n",
    "            \"x_coordinate\",\n",
    "            \"y_coordinate\",\n",
    "        ]\n",
    "    ]\n",
    "    return df\n",
    "\n",
    "\n",
    "def parse_stop_event(df):\n",
    "\n",
    "    month_dict = {\n",
    "        \"JAN\": \"01\",\n",
    "        \"FEB\": \"02\",\n",
    "        \"MAR\": \"03\",\n",
    "        \"APR\": \"04\",\n",
    "        \"MAY\": \"05\",\n",
    "        \"JUN\": \"06\",\n",
    "        \"JUL\": \"07\",\n",
    "        \"AUG\": \"08\",\n",
    "        \"SEP\": \"09\",\n",
    "        \"OCT\": \"10\",\n",
    "        \"NOV\": \"11\",\n",
    "        \"DEC\": \"12\",\n",
    "    }\n",
    "\n",
    "    @lru_cache()\n",
    "    def parse_date(date_str):\n",
    "        date = date_str.split(\":\")[0]\n",
    "        day = date[:2]\n",
    "        month = month_dict[date[2:5]]\n",
    "        year = date[5:]\n",
    "        return year + \"/\" + month + \"/\" + day\n",
    "\n",
    "    df[\"service_date\"] = df[\"service_date\"].apply(parse_date)\n",
    "\n",
    "    df[\"service_date\"] = pd.to_datetime(df[\"service_date\"], format=\"%Y/%m/%d\")\n",
    "\n",
    "    df[\"day_of_year\"] = df[\"service_date\"].dt.day_of_year\n",
    "    df[\"day_of_week\"] = df[\"service_date\"].dt.day_of_week\n",
    "\n",
    "    df[\"arrival_deviance\"] = df[\"stop_time\"] - df[\"arrive_time\"]\n",
    "    df[\"arrive_deviance_departure_delta\"] = (\n",
    "        df[\"arrival_deviance\"] + df[\"leave_time\"] - df[\"arrive_time\"]\n",
    "    )\n",
    "\n",
    "    minutes_per_time_cat = 5\n",
    "    times = [\"stop_time\", \"arrive_time\", \"leave_time\"]\n",
    "    time_cats = [\"time_cat_\" + x for x in times]\n",
    "    df[time_cats] = df[times] // 60 // minutes_per_time_cat\n",
    "\n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "70d246f8-53c3-4583-a1b1-ba8f12d2a12c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on 2 trimet_stop_event - Fall 2019.csv\n",
      "working on 2 trimet_stop_event - Mar to Aug 2020.csv\n",
      "working on 2 trimet_stop_event - Spring 2019.csv\n",
      "working on 2 trimet_stop_event - Summer 2019.csv\n",
      "working on 2 trimet_stop_event - Winter 2018.csv\n",
      "working on 2 trimet_stop_event - Winter 2019-20.csv\n",
      "working on trimet_stop_event - Fall 2018 v2.csv\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "                <audio  controls=\"controls\" autoplay=\"autoplay\">\n",
       "                    <source src=\"https://www.myinstants.com/media/sounds/magic_immune.mp3\" type=\"audio/mpeg\" />\n",
       "                    Your browser does not support the audio element.\n",
       "                </audio>\n",
       "              "
      ],
      "text/plain": [
       "<IPython.lib.display.Audio object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "stop_events = os.path.join(data_dir, \"stop_event\")\n",
    "\n",
    "stop_event_file_names = [\n",
    "    \"2 trimet_stop_event - Fall 2019.csv\",\n",
    "    \"2 trimet_stop_event - Mar to Aug 2020.csv\",\n",
    "    \"2 trimet_stop_event - Spring 2019.csv\",\n",
    "    \"2 trimet_stop_event - Summer 2019.csv\",\n",
    "    \"2 trimet_stop_event - Winter 2018.csv\",\n",
    "    \"2 trimet_stop_event - Winter 2019-20.csv\",\n",
    "    \"trimet_stop_event - Fall 2018 v2.csv\",\n",
    "]\n",
    "\n",
    "\n",
    "stop_event_dfs = []\n",
    "for file_name in stop_event_file_names:\n",
    "    print(\"working on\", file_name)\n",
    "    file_name = os.path.join(stop_events, file_name)\n",
    "    df = dd.read_csv(\n",
    "        file_name,\n",
    "        dtype={\n",
    "            \"LOCATION_DISTANCE\": \"float32\",\n",
    "            \"PATTERN_DISTANCE\": \"float32\",\n",
    "            \"TRAIN_MILEAGE\": \"float32\",\n",
    "            \"X_COORDINATE\": \"float32\",\n",
    "            \"Y_COORDINATE\": \"float32\",\n",
    "            \"DATA_SOURCE\": \"float64\",\n",
    "            \"LOCATION_ID\": \"float64\",\n",
    "            \"SCHEDULE_STATUS\": \"float64\",\n",
    "        },\n",
    "    )\n",
    "    df = lower_case_sort_columns(df)\n",
    "    df = df[df[\"route_number\"] == 9].compute()\n",
    "    df = parse_stop_event(df)\n",
    "    stop_event_dfs.append(df)\n",
    "\n",
    "df = pd.concat(stop_event_dfs, axis=0, ignore_index=True)\n",
    "\n",
    "times = [\"stop_time\", \"arrive_time\", \"leave_time\"]\n",
    "time_cats = [\"time_cat_\" + x for x in times]\n",
    "categories = [\n",
    "    \"vehicle_number\",\n",
    "    \"train\",\n",
    "    \"route_number\",\n",
    "    \"direction\",\n",
    "    \"service_key\",\n",
    "    \"location_id\",\n",
    "    \"door\",\n",
    "    \"lift\",\n",
    "    \"day_of_year\",\n",
    "    \"day_of_week\",\n",
    "    *time_cats,\n",
    "]\n",
    "\n",
    "df[categories] = df[categories].astype(\"category\")\n",
    "\n",
    "df = df.sort_values([\"service_date\", \"train\", \"trip_number\", \"stop_time\"])\n",
    "df = df.reset_index()\n",
    "\n",
    "save_file = os.path.join(data_dir, \"mega_stop_event_1.hdf\")\n",
    "df.to_hdf(save_file, \"/df\", complevel=1, format=\"table\")\n",
    "\n",
    "allDone()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "79f73c56-f880-4b65-8541-d9a0d62551f1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "                <audio  controls=\"controls\" autoplay=\"autoplay\">\n",
       "                    <source src=\"https://www.myinstants.com/media/sounds/magic_immune.mp3\" type=\"audio/mpeg\" />\n",
       "                    Your browser does not support the audio element.\n",
       "                </audio>\n",
       "              "
      ],
      "text/plain": [
       "<IPython.lib.display.Audio object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# extra clean up\n",
    "\n",
    "save_file = os.path.join(data_dir, \"mega_stop_event.hdf\")\n",
    "df = pd.read_hdf(save_file, \"/df\")\n",
    "\n",
    "# df = df.drop(columns=[\"location_distance\", \"delta_train_mileage\", \"delta_pattern_distance\"])\n",
    "# df = df.drop(columns=[\"apc_status\"])\n",
    "import util\n",
    "\n",
    "# df = df.sort_values([\"service_date\", \"trip_number\", \"arrive_time\"])\n",
    "# util.big_print(df.head(1000))\n",
    "# df = df.sort_values([\"trip_number\", \"arrive_time\", \"service_date\"])\n",
    "\n",
    "\n",
    "# save_file = os.path.join(data_dir, \"mega_stop_event.hdf\")\n",
    "# df.to_hdf(save_file, \"/df\", append=False, complevel=1, format=\"table\")\n",
    "\n",
    "allDone()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "b028ab84-7817-4f80-b774-28932ac10645",
   "metadata": {},
   "outputs": [],
   "source": [
    "# df.to_hdf(save_file, \"/df\", append=False, complevel=1, format=\"t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "19360526-43ac-4e28-be4e-2f63be862f74",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(1266369, 31)"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <zmq.eventloop.ioloop.ZMQIOLoop object at 0x000002D92BE29F70>>, <Task finished name='Task-1694' coro=<Client._update_scheduler_info() done, defined at C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py:1198> exception=MemoryError('Unable to allocate internal buffer.')>)\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 741, in _run_callback\n",
      "    ret = callback()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 765, in _discard_future_result\n",
      "    future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1202, in _update_scheduler_info\n",
      "    self._scheduler_identity = SchedulerInfo(await self.scheduler.identity())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 866, in send_recv_from_rpc\n",
      "    result = await send_recv(comm=comm, op=key, **kwargs)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 649, in send_recv\n",
      "    await comm.write(msg, serializers=serializers, on_error=\"raise\")\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.batched - ERROR - Error in batched write\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\batched.py\", line 93, in _background_send\n",
      "    nbytes = yield self.comm.write(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\gen.py\", line 762, in run\n",
      "    value = future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <zmq.eventloop.ioloop.ZMQIOLoop object at 0x000002D92BE29F70>>, <Task finished name='Task-1732' coro=<Client._update_scheduler_info() done, defined at C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py:1198> exception=MemoryError('Unable to allocate internal buffer.')>)\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 741, in _run_callback\n",
      "    ret = callback()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 765, in _discard_future_result\n",
      "    future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1202, in _update_scheduler_info\n",
      "    self._scheduler_identity = SchedulerInfo(await self.scheduler.identity())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 866, in send_recv_from_rpc\n",
      "    result = await send_recv(comm=comm, op=key, **kwargs)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 649, in send_recv\n",
      "    await comm.write(msg, serializers=serializers, on_error=\"raise\")\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <zmq.eventloop.ioloop.ZMQIOLoop object at 0x000002D92BE29F70>>, <Task finished name='Task-1813' coro=<Client._update_scheduler_info() done, defined at C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py:1198> exception=MemoryError('Unable to allocate internal buffer.')>)\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 741, in _run_callback\n",
      "    ret = callback()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 765, in _discard_future_result\n",
      "    future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1202, in _update_scheduler_info\n",
      "    self._scheduler_identity = SchedulerInfo(await self.scheduler.identity())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 866, in send_recv_from_rpc\n",
      "    result = await send_recv(comm=comm, op=key, **kwargs)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 649, in send_recv\n",
      "    await comm.write(msg, serializers=serializers, on_error=\"raise\")\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.batched - ERROR - Error in batched write\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\batched.py\", line 93, in _background_send\n",
      "    nbytes = yield self.comm.write(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\gen.py\", line 762, in run\n",
      "    value = future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <zmq.eventloop.ioloop.ZMQIOLoop object at 0x000002D92BE29F70>>, <Task finished name='Task-1895' coro=<Client._update_scheduler_info() done, defined at C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py:1198> exception=MemoryError('Unable to allocate internal buffer.')>)\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 741, in _run_callback\n",
      "    ret = callback()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 765, in _discard_future_result\n",
      "    future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1202, in _update_scheduler_info\n",
      "    self._scheduler_identity = SchedulerInfo(await self.scheduler.identity())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 866, in send_recv_from_rpc\n",
      "    result = await send_recv(comm=comm, op=key, **kwargs)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 649, in send_recv\n",
      "    await comm.write(msg, serializers=serializers, on_error=\"raise\")\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <zmq.eventloop.ioloop.ZMQIOLoop object at 0x000002D92BE29F70>>, <Task finished name='Task-1976' coro=<Client._update_scheduler_info() done, defined at C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py:1198> exception=MemoryError('Unable to allocate internal buffer.')>)\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 741, in _run_callback\n",
      "    ret = callback()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 765, in _discard_future_result\n",
      "    future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1202, in _update_scheduler_info\n",
      "    self._scheduler_identity = SchedulerInfo(await self.scheduler.identity())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 866, in send_recv_from_rpc\n",
      "    result = await send_recv(comm=comm, op=key, **kwargs)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 649, in send_recv\n",
      "    await comm.write(msg, serializers=serializers, on_error=\"raise\")\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <zmq.eventloop.ioloop.ZMQIOLoop object at 0x000002D92BE29F70>>, <Task finished name='Task-2049' coro=<Client._update_scheduler_info() done, defined at C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py:1198> exception=MemoryError('Unable to allocate internal buffer.')>)\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 741, in _run_callback\n",
      "    ret = callback()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 765, in _discard_future_result\n",
      "    future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1202, in _update_scheduler_info\n",
      "    self._scheduler_identity = SchedulerInfo(await self.scheduler.identity())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 866, in send_recv_from_rpc\n",
      "    result = await send_recv(comm=comm, op=key, **kwargs)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 649, in send_recv\n",
      "    await comm.write(msg, serializers=serializers, on_error=\"raise\")\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 232, in write\n",
      "    frames = await to_frames(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 51, in to_frames\n",
      "    return _to_frames()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client\n",
      "distributed.protocol.core - CRITICAL - Failed to Serialize\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.comm.utils - ERROR - Unable to allocate internal buffer.\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\utils.py\", line 31, in _to_frames\n",
      "    protocol.dumps(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\protocol\\core.py\", line 70, in dumps\n",
      "    frames[0] = msgpack.dumps(msg, default=_encode_default, use_bin_type=True)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\msgpack\\__init__.py\", line 35, in packb\n",
      "    return Packer(**kwargs).pack(o)\n",
      "  File \"msgpack/_packer.pyx\", line 120, in msgpack._cmsgpack.Packer.__cinit__\n",
      "MemoryError: Unable to allocate internal buffer.\n",
      "distributed.utils - ERROR - Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 193, in read\n",
      "    frames_nbytes = await stream.read_bytes(fmt_size)\n",
      "tornado.iostream.StreamClosedError: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 321, in connect\n",
      "    handshake = await asyncio.wait_for(comm.read(), time_left())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\asyncio\\tasks.py\", line 494, in wait_for\n",
      "    return fut.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 203, in read\n",
      "    convert_stream_closed_error(self, e)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 125, in convert_stream_closed_error\n",
      "    raise CommClosedError(f\"in {obj}: {exc}\") from exc\n",
      "distributed.comm.core.CommClosedError: in <closed TCP>: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\utils.py\", line 638, in log_errors\n",
      "    yield\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1410, in _close\n",
      "    await self.cluster.close()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 419, in _close\n",
      "    await self._correct_state()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 334, in _correct_state_internal\n",
      "    await self.scheduler_comm.retire_workers(workers=list(to_close))\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 795, in send_recv_from_rpc\n",
      "    comm = await self.live_comm()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 753, in live_comm\n",
      "    comm = await connect(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 326, in connect\n",
      "    raise OSError(\n",
      "OSError: Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s\n",
      "distributed.utils - ERROR - Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 193, in read\n",
      "    frames_nbytes = await stream.read_bytes(fmt_size)\n",
      "tornado.iostream.StreamClosedError: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 321, in connect\n",
      "    handshake = await asyncio.wait_for(comm.read(), time_left())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\asyncio\\tasks.py\", line 494, in wait_for\n",
      "    return fut.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 203, in read\n",
      "    convert_stream_closed_error(self, e)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 125, in convert_stream_closed_error\n",
      "    raise CommClosedError(f\"in {obj}: {exc}\") from exc\n",
      "distributed.comm.core.CommClosedError: in <closed TCP>: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\utils.py\", line 638, in log_errors\n",
      "    yield\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1136, in _reconnect\n",
      "    await self._close()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1410, in _close\n",
      "    await self.cluster.close()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 419, in _close\n",
      "    await self._correct_state()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 334, in _correct_state_internal\n",
      "    await self.scheduler_comm.retire_workers(workers=list(to_close))\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 795, in send_recv_from_rpc\n",
      "    comm = await self.live_comm()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 753, in live_comm\n",
      "    comm = await connect(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 326, in connect\n",
      "    raise OSError(\n",
      "OSError: Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s\n",
      "distributed.utils - ERROR - Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 193, in read\n",
      "    frames_nbytes = await stream.read_bytes(fmt_size)\n",
      "tornado.iostream.StreamClosedError: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 321, in connect\n",
      "    handshake = await asyncio.wait_for(comm.read(), time_left())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\asyncio\\tasks.py\", line 494, in wait_for\n",
      "    return fut.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 203, in read\n",
      "    convert_stream_closed_error(self, e)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 125, in convert_stream_closed_error\n",
      "    raise CommClosedError(f\"in {obj}: {exc}\") from exc\n",
      "distributed.comm.core.CommClosedError: in <closed TCP>: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\utils.py\", line 638, in log_errors\n",
      "    yield\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1283, in _handle_report\n",
      "    await self._reconnect()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1136, in _reconnect\n",
      "    await self._close()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1410, in _close\n",
      "    await self.cluster.close()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 419, in _close\n",
      "    await self._correct_state()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 334, in _correct_state_internal\n",
      "    await self.scheduler_comm.retire_workers(workers=list(to_close))\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 795, in send_recv_from_rpc\n",
      "    comm = await self.live_comm()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 753, in live_comm\n",
      "    comm = await connect(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 326, in connect\n",
      "    raise OSError(\n",
      "OSError: Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s\n",
      "tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <zmq.eventloop.ioloop.ZMQIOLoop object at 0x000002D92BE29F70>>, <Task finished name='Task-2092' coro=<SpecCluster._correct_state_internal() done, defined at C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py:327> exception=OSError('Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s')>)\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 193, in read\n",
      "    frames_nbytes = await stream.read_bytes(fmt_size)\n",
      "tornado.iostream.StreamClosedError: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 321, in connect\n",
      "    handshake = await asyncio.wait_for(comm.read(), time_left())\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\asyncio\\tasks.py\", line 494, in wait_for\n",
      "    return fut.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 203, in read\n",
      "    convert_stream_closed_error(self, e)\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\tcp.py\", line 125, in convert_stream_closed_error\n",
      "    raise CommClosedError(f\"in {obj}: {exc}\") from exc\n",
      "distributed.comm.core.CommClosedError: in <closed TCP>: Stream is closed\n",
      "\n",
      "The above exception was the direct cause of the following exception:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 741, in _run_callback\n",
      "    ret = callback()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\tornado\\ioloop.py\", line 765, in _discard_future_result\n",
      "    future.result()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1283, in _handle_report\n",
      "    await self._reconnect()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1136, in _reconnect\n",
      "    await self._close()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\client.py\", line 1410, in _close\n",
      "    await self.cluster.close()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 419, in _close\n",
      "    await self._correct_state()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\deploy\\spec.py\", line 334, in _correct_state_internal\n",
      "    await self.scheduler_comm.retire_workers(workers=list(to_close))\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 795, in send_recv_from_rpc\n",
      "    comm = await self.live_comm()\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\core.py\", line 753, in live_comm\n",
      "    comm = await connect(\n",
      "  File \"C:\\tools\\miniconda3\\envs\\tf\\lib\\site-packages\\distributed\\comm\\core.py\", line 326, in connect\n",
      "    raise OSError(\n",
      "OSError: Timed out during handshake while connecting to tcp://127.0.0.1:54401 after 10 s\n"
     ]
    }
   ],
   "source": [
    "df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8015c449-3870-4794-a7ba-ce26246efb4a",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "tf",
   "language": "python",
   "name": "tf"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.10"
  },
  "toc-autonumbering": false,
  "toc-showcode": true,
  "toc-showmarkdowntxt": false,
  "toc-showtags": false
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
